appveyor.yml

```diff
@@ -67,13 +67,13 @@ for:
     
     before_build:
       - sh: >-
-          if [ -z "$APPVEYOR_REPO_TAG_NAME" ]; then export SUFFIX=$(git describe --tags); else export SUFFIX=$APPVEYOR_REPO_TAG_NAME-${APPVEYOR_REPO_COMMIT:0:7}; fi
+          if [ -z "$APPVEYOR_REPO_TAG_NAME" ]; then export SUFFIX=$(git describe --tags | sed -e 's/-/+/'); else export SUFFIX=$APPVEYOR_REPO_TAG_NAME-${APPVEYOR_REPO_COMMIT:0:7}; fi
```

Comment:
`v0.7.0-2-g01c9e00f` would become `v0.7.0+2-g01c9e00f` now, so that it can connect to the server with the semver checking, easier for testing the build from CI.